monkey/common/agent_event_serializers/register.py

```diff
@@ -2,6 +2,7 @@
     AgentShutdownEvent,
     CredentialsStolenEvent,
     ExploitationEvent,
+    FileEncryptedEvent,
```

Comment:
Looks like we already broke consistency. If there are `ExploitationEvent`, `PasswordResorationEvent`, then this should probably be `FileEncryptionEvent`